source/Makefile.vars

```diff
@@ -44,7 +44,7 @@ CEREAL_DIR    = /usr/local/include/cereal
 ## CEREAL_DIR        should contain an include folder.
 ##---------------------------------------------------------------------
 
-# FFTW_DIR = /public/soft/fftw_3.3.8
+# FFTW_DIR = /root/lib/FFTW3
```

Comment:
same question as ELPA. Why do modify vars about FFTW?

Reply:
you are right, these changes are unnecessary and was made by the former contributor

Reply:
what should the new vars for new libs be? (pexsi, parmetis, superlu_dist)